embassy-stm32/build.rs

```diff
@@ -308,6 +308,10 @@ fn main() {
     // ========
     // Generate RccPeripheral impls
 
+    // TODO: maybe get this from peripheral kind? Not sure
+    let refcounted_peripherals = HashSet::from(["UART", "USART", "SPI", "I2C"]);
```

Comment:
why refcount spi, i2c? these have no split.
also LPUARTs should be refcounted I think

Reply:
lpuarts are already included